roles/openshift_examples/files/examples/x86_64/xpaas-streams/eap-cd-image-stream.json

```
@@ -9,29 +9,29 @@
         }
     },
     "items": [
-        {
+    {
             "kind": "ImageStream",
             "apiVersion": "v1",
             "metadata": {
                 "name": "eap-cd-openshift",
                 "annotations": {
                     "openshift.io/display-name": "JBoss EAP continuous delivery",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "version": "13.0"
+                    "version": "14.0"
```

Use latest tag version here?
```suggestion
                    "version": "15.0"
```